dias/analyzers/sensitivity_analyzer.py

```diff
@@ -178,26 +163,19 @@ def run(self):
         """Task stage: analyzes data from the last period."""
         stop_time = datetime.utcnow() - self.lag
 
-        # Refresh the database
-        self.refresh_data_index()
-
-        cursor = self.data_index.cursor()
-        query = "SELECT stop FROM files ORDER BY stop DESC LIMIT 1"
-        results = list(cursor.execute(query))
-        start_time = results[0][0] if results else stop_time - self.period
-
         # Find all calibration files
-        file_list = self.find_all_archive(
-            instrument=self.instrument, data_product=self.acq_suffix
-        )
-        file_list = self.filter_files_by_time(file_list, start_time, stop_time)
+        file_list = self.new_files(filetypes=instrument)
+        start_time = stop_time - self.period
```

Comment:
start time is set but not used

Reply:
Why did this get resolved?

Reply:
Oh gosh, I don't remember. I remember erasing this, and resolving, but it is possible among all-the-things, that the change got lost. I will erase it.